compiler/src/iree/compiler/Codegen/Common/GPU/AMDGPUDistributeContract.cpp

```diff
@@ -211,7 +217,7 @@ struct DistributeContract final : OpDistributionPattern<vector::ContractionOp> {
     applyPermutationToVector(rhsOffsets, rhsLayout.getBatchOrder());
   }
 
-  struct MFMAParameters {
+  struct MMAParameters {
```

Comment:
These parameters are MFMA/WMMA specific. Better to name it as AMDMMAParameters or create a separate WMMAParameters struct.

Reply:
renamed to AMDMMAParameters. :)